src/db.c

```diff
@@ -5103,6 +5106,7 @@ queue_enum_fetch(struct query_params *qp, struct db_queue_item *queue_item, int
   queue_item->track = sqlite3_column_int(qp->stmt, 20);
   queue_item->disc = sqlite3_column_int(qp->stmt, 21);
   queue_item->artwork_url = strdup_if((char *)sqlite3_column_text(qp->stmt, 22), keep_item);
+  queue_item->composer = strdup_if((char *)sqlite3_column_text(qp->stmt, 24), keep_item);
```

Comment:
Does this work? Should it not be `23`?

Reply:
column 23 is `queue_version` .  I add the column here and it becomes col 24; the full definition is in `db_init.c`

https://github.com/ejurgensen/forked-daapd/pull/619/files#diff-8fc831350c1c371efb7094030a5b5ff2

I did notice that the previous `queue` table updates took 2 different approaches:
* add the column (see `U_V1903_...`)
* drop the `queue` tbl entirely and recreate (see  `U_V1906_...`)

I went with the first approach;  this was tested upgrading an existing db (resulting in the alter tb and addition of column) as well as having no db file (creates via the `db_init`)

Reply:
Thanks, i missed that the queue version is missing. Doing it with an alter statement is correct. Dropping a table should only be done if an alter table is not possible. 

@ejurgensen imo this is good for merge.